.github/workflows/ci.yml

```diff
@@ -0,0 +1,77 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  should-skip:
+    continue-on-error: true
+    runs-on: ubuntu-latest
```

Comment:
we were seeing some issues running on ubuntu-latest and have pinned to ubuntu-18.04 for the time being
```suggestion
    runs-on: ubuntu-18.04
```